MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -37,6 +39,14 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		tableModel = cellControllers
 	}
 
+	func display(_ viewModel: FeedErrorViewModel) {
+		guard let message = viewModel.message else {
+			errorView?.hideMessage()
+			return
+		}
+		errorView?.show(message: message)
```

Comment:
Suggestion: 

An `if/else` statement is usually easier to read/follow in such cases:

"If there's a message to show, show it. Else, hide the existing one."

```suggestion
		if let message = viewModel.message {
			errorView?.show(message: message)
		} else {
			errorView?.hideMessage()
		}
```

Instead of "guard against not having a message. Else, hide the existing one. Show the message if it passed the guard statement."